components/raftstore/src/store/msg.rs

```diff
@@ -269,6 +270,7 @@ pub enum CasualMessage<EK: KvEngine> {
         // TODO: support meta key.
         split_keys: Vec<Vec<u8>>,
         callback: Callback<EK::Snapshot>,
+        source: Cow<'static, str>,
```

Comment:
```suggestion
        source: &'static, str,
```
How about consistent with `HalfSplitRegion`

Reply:
It needs to accept ip address from grpc, which is a string.